### ISSUE TITLE
[Feature request] Improve numeric data type

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1725]
**I'm submitting a ...**
[x] feature request

**...about** 
[x] Content Types or data management

**Current behavior**
This is a follow up on this issue:
[All numeric fields used as decimals in template](https://github.com/2sic/2sxc/issues/1663)
Number data type only accepts one (sub-)type, that is later used as a decimal in templates.

**Expected behavior**
Number itself could be divided in int, double, etc. This would make views and controllers muck easier to create.

Also, nothing should prevent a numeric field from being used as a dropdown or even a dropdown from query. Doesn't it make sense to link a dropdown query value to a numeric ID from other table? And keeping it numeric? Or a simple dropdown with a numeric id and string label. Right now I'm forced to convert the numeric id to a string field, and then numeric again in the controller/view.

Just some thoughts, and probably very low priority, but it would improve data management.

**Comment by iJungleboy** [comment]
This is fairly low priority, but creating a number-dropdown will probably be on our todo list once angular 7/8 UI is done.

**Comment by iJungleboy** [comment]
BTW: Check out #2467 - we just created a number-dropdown field

I believe this solves part of the wish above.

**Comment by iJungleboy** [comment]
number-Dropdown works as of now. 
Number from query not yet, but I hope it will be part of an upcoming release where we provide a dropdown from various sources. 
As such this won't have much activity, so I'll close this for now.
